*** /root/rpm-spec-mode.el	2003-04-21 22:43:48.000000000 +0200
--- /usr/share/emacs/site-lisp/rpm-spec-mode.el	2003-04-21 22:59:13.000000000 +0200
***************
*** 530,535 ****
--- 530,538 ----
  
  ;;------------------------------------------------------------
  
+ (add-hook 'rpm-spec-mode-new-file-hook 'rpm-spec-initialize)
+ 
+ 
  ;;;###autoload
  (defun rpm-spec-mode ()
    "Major mode for editing RPM spec files.
***************
*** 558,564 ****
    (easy-menu-add rpm-spec-mode-menu)
  
    (if (= (buffer-size) 0)
!       (rpm-spec-initialize))
  
    (if (executable-find "rpmbuild")
        (setq rpm-spec-build-command "rpmbuild")
--- 561,567 ----
    (easy-menu-add rpm-spec-mode-menu)
  
    (if (= (buffer-size) 0)
!       (run-hooks 'rpm-spec-mode-new-file-hook))
  
    (if (executable-find "rpmbuild")
        (setq rpm-spec-build-command "rpmbuild")
